### ISSUE TITLE
Migrate to postgres

### ISSUE THREAD

**Issue by sebbacon** [opensafely-core/job-server#1033]
* We all have experience of postgres
* At some point we expect we will repeatedly start hitting the practical concurrency limits of sqlite
* Most of the dev work will be to work around `n+1` issues with `select_related` and friends. 
  * @ghickman suggested "I genuinely think just getting all the URLs from python manage show_urls and having python open then in your browser with DJDT configured to pre-open would take 50% of the work away"
* We think `JobRequestList` views are likely to be the pathological case to test against

**Comment by ghickman** [comment]
pgloader appears to successfully migrate all the data into postgres with this work file:```load database
     from sqlite://db.sqlite3
     into postgresql://localhost/jobserver

 with include drop, create tables, create indexes, reset sequences

  set work_mem to '16MB', maintenance_work_mem to '512 MB';
```

The event log page is showing 83 queries in 93ms locally, most of those queries are the expected duplicates from the JobRequest properties not making use of prefetch_selected jobs.

The home page is showing a similar issue for the 5 JobRequests listed there.

**Comment by inglesp** [comment]
You don't want pgloader to create the tables for you!  It won't create all the constraints the Django expects, and then you'll have trouble down the line.  Better is to run migrations against an empty postgres database, and then to use `with data only`.

</hard-won-experience>

**Comment by ghickman** [comment]
Great to know 👍

I suspect we want to keep reset sequences so our PKs don't overlap?

**Comment by inglesp** [comment]
> I suspect we want to keep reset sequences so our PKs don't overlap?

Good point!

**Comment by StevenMaude** [comment]
It seems like we've switched to PostgreSQL now. So can this issue be closed?

One small clean-up task I previously noted: we still have `litecli` in dependencies, which could be removed if we've completed the migration definitively. Removing that would also shrinks our list of dependencies slightly as there are a few in there only used by `litecli`.

One clean-up task I previously didn't: the `sqlite.load` file in the root of the repository could be removed, again if we've completed migration.
